src/resources/ble_esp32.cc

```diff
@@ -546,8 +533,7 @@ PRIMITIVE(init) {
 
   ble_hs_cfg.sync_cb = ble_on_sync;
 
-  group->register_resource(gap);
-  group->set_gap(gap);
+  nimble_port_init();
```

Comment:
Should we deinit if setting the server config fails?

Reply:
The BLEResourceGroup destructor does that for us.